stdlib/inspect.pyi

```diff
@@ -345,12 +345,12 @@ class Signature:
 
 if sys.version_info >= (3, 10):
     def get_annotations(
-        obj: Callable[..., object] | type[Any] | ModuleType,
+        obj: Callable[..., object] | type[Any] | ModuleType,  # any callable, class, or module
```

Comment:
I think `type[object]` is just as good here, it allows the same set of types in the context of a parameter annotation

```suggestion
        obj: Callable[..., object] | type[object] | ModuleType,  # any callable, class, or module
```